monkey/infection_monkey/config.py

```diff
@@ -259,23 +277,22 @@ def get_exploit_user_password_or_hash_product(self):
 
     extract_azure_creds = True
 
+    ###########################
+    # post breach actions
+    ###########################
     post_breach_actions = []
     custom_PBA_linux_cmd = ""
     custom_PBA_windows_cmd = ""
     PBA_linux_filename = None
     PBA_windows_filename = None
 
-    @staticmethod
-    def hash_sensitive_data(sensitive_data):
-        """
-        Hash sensitive data (e.g. passwords). Used so the log won't contain sensitive data plain-text, as the log is
-        saved on client machines plain-text.
+    ###########################
+    # testing configuration
+    ###########################
+    export_monkey_telems = False
 
-        :param sensitive_data: the data to hash.
-        :return: the hashed data.
-        """
-        password_hashed = hashlib.sha512(sensitive_data.encode()).hexdigest()
-        return password_hashed
+    def get_hop_count(self):
+        return self.max_depth - self.depth
```

Comment:
I'd love if we could rename this function in a more descriptive manner. 
```suggestion
    def get_how_many_hops_from_island(self):
        return self.max_depth - self.depth
```
or something like that